CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 ## Unreleased
 
 * Add blue arrow option to action link ([PR #2330](https://github.com/alphagov/govuk_publishing_components/pull/2330))
```

Comment:
This will need a rebase now - this change has been released: https://github.com/alphagov/govuk_publishing_components/pull/2332